.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-96c96ce89d2731846b733054f3cef8915f31a609
+f2b36df6a1a80137eff7644e6d0f4eeb7ff429d6
```

Comment:
Is the xla update necessary here?

Reply:
yes, see https://github.com/pytorch/xla/pull/4067